scripts/migrate.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable camelcase */
+/* eslint-disable new-cap */
+/* eslint-disable no-multi-str */
+/* eslint-disable no-template-curly-in-string */
+
+require("dotenv").config()
+
+const settings = require("../migrations/settings")
+
+if (process.argv.length < 3) {
+  console.log()
+  console.log("\n\
+    Usage: npm run migrate <[Realm.]Network>\n\n\
+  ")
+  process.exit(0)
+}
+
+const rn = require("./utils").getRealmNetworkFromString(process.argv[2])
+const realm = rn[0]; const network = rn[1]
+
+if (!settings.networks[realm] || !settings.networks[realm][network]) {
+  console.error(`\n!!! Network "${network}" not found.\n`)
+  if (settings.networks[realm]) {
+    console.error(`> Available networks in realm "${realm}":`)
+    console.error(settings.networks[realm])
+  } else {
+    console.error("> Available networks:")
+    console.error(settings.networks)
+  }
+  process.exit(1)
+}
+
+migrate(network)
+
+/// /////////////////////////////////////////////////////////////////////////////
+
+async function migrate (network) {
+  console.log(`> Migrating from ${process.env.FLATTENED_DIRECTORY} into network '${network}'...`)
+  await new Promise((resolve) => {
```

Comment:
It doesn't affect if you add `await` or not. But returning the promise you are allowing anyone to use `await` or `.then`
```suggestion
  return new Promise((resolve) => {
```

Reply:
Right you are